packages/destination-actions/src/destinations/salesforce/__tests__/custom.test.ts

```diff
@@ -0,0 +1,62 @@
+import nock from 'nock'
+import { createTestEvent, createTestIntegration } from '@segment/actions-core'
+import Destination from '../index'
+import { API_VERSION } from '../sf-operations'
+
+const testDestination = createTestIntegration(Destination)
+
+const settings = {
+  instanceUrl: 'https://test.com'
+}
+
+describe('Salesforce', () => {
+  describe('Custom', () => {
```

Comment:
What do you think about being more specific in this description?
```suggestion
  describe('`Custom Object Tests`', () => {
```